test/test_jit_cuda_fuser.py

```diff
@@ -469,21 +471,24 @@ def t(x: torch.Tensor, y: torch.Tensor, z: torch.Tensor):
         self.assertGraphContains(t_jit.graph_for(x, y, z), FUSION_GUARD)
 
     def _unary_test_helper(self, operation, dtype, random_data):
-        shape = (4, 8, 32, 32)
+        gradient_check = (dtype == torch.float64) and random_data
+        shape = (8, 7)
```

Comment:
Any particular reason for changing the test shape?

Reply:
The shape is to avoid OOM memory during `gradcheck`.